src/validation-function.js

```diff
@@ -63,6 +63,8 @@ class ValidationFunction {
         args.push('DummyShapesGraph')
       } else if (arg === 'this') {
         args.push(focusNode)
+      } else if (arg === 'context') {
+        args.push(this.context)
```

Comment:
Injecting the context is a short-term solution to make it work. I will remove the whole dependency injection mechanism in a separate PR. Every validation function should probably take the same set of arguments. I may even put then in a class that already has access to the context.